### ISSUE TITLE
Text selection doesn't happen if you click outside the text lines

### ISSUE THREAD

**Issue by ApostolosB** [Obsidian-StudiosInc/ecrire#31]
##### Description
Description of issue
You click outside the lines of text and moving the pointer to the lines of text doesn't turn them blue.
##### Steps to reproduce
1. Open a file with multiple text lines.
2. Click outside (below) the last text line and drag your cursor to select the text 
3. Nothing happens.
...

##### System information
- EFL Version = 1.20.7 & git
- E Version = 0.22.1 & git
- Wayland

**Comment by wltjr** [comment]
I see what you mean here, thanks for the steps and break down. Basically, with line numbers on elm_code, if you do not click on a numbered line, it does not select text. That maybe by design, not sure.

Another way to say, based on coloring. if you do not click within the gray text area. It will not select text. If you click on the black area, and then enter the gray area. It does not select the text. If you click within the gray area, you can select text.

Basically you MUST click in the gray area, lines of text, to select text. If that is not by design, then will need to do something in the black area to add a click listener and start selection on the last line in gray area.

I am not sure who will be maintaining elm_code, or fixing issues. Seems like active maintainer is stepping back, and not sure if others will further or address issues.

**Comment by wltjr** [comment]
There is another issue that is related, and could use another issue/task. Right click context menus only show in the gray area/lines of text, not in the black area. Other editors seem to show the context menu anywhere in the editor area regardless of the amount of lines of text. Like selection they seem to end up on last line.

In leafpad for example. I can have 10 lines, and empty area beneath. I can click beneath the 10 lines and drag up to start selection. Just the same I can right click beneath the lines and paste. Which puts the data start on line 11.

The editor is missing function when not in the gray area/lines of text. Both selection and right click context menu.

Also if you do have text selected and right click in the black area, rather than gray area/lines of text. It will deselect the selected text. Which is likely not ideal either. That should remain selected, and show right click context menu, cut, copy, paste, cancel.

**Comment by ApostolosB** [comment]
upstream > https://phab.enlightenment.org/T6773

**Comment by wltjr** [comment]
Nice thank you! May need one for the context stuff as well, right click menu. Hopefully someday they will re-enable my phab account... Much appreciated since I cannot file upstream!!!

**Comment by wltjr** [comment]
Just realized my color description is specific to my theme [eminence](https://github.com/Obsidian-StudiosInc/eminence).
![shot-2018-03-13_19-32-54](https://user-images.githubusercontent.com/12835340/37375348-83aa59a4-26f5-11e8-8964-b3e22d7ab9ad.jpg)
I need to make it all one color and other things
